src/Index/AbstractAggregateIndex.php

```diff
@@ -42,6 +42,9 @@ protected function registerIndex(ReadableIndex $index)
         $index->on('definition-added', function () {
             $this->emit('definition-added');
         });
+        $index->on('wipe', function() {
```

Comment:
Who subscribes to this event? What is it needed for?
As it is implemented, it will be triggered many times for a single `wipe()` call on an AggregateIndex.